static/js/theme-map/VerticalFullPageMapOrchestrator.js

```diff
@@ -478,14 +494,42 @@ class VerticalFullPageMapOrchestrator extends ANSWERS.Component {
     }
   }
 
+  /**
+   * Update the map wrapper's "top" and "height" css properties so that it is not covered up
+   * by the no results block in mobile. 
+   *
+   * @param {ResultsContext} resultsContext 
+   */
+  updateMapDimensionsForNoResults(resultsContext) {
+    if (!this.isInMobileMapView) {
+      return;
+    }
+    const noResults = this._noResultsContainer;
+    const map = document.querySelector(this._mapContainerSelector);
+    if (!noResults || !map) {
+      return;
+    }
+    if (resultsContext === ResultsContext.NO_RESULTS) {
+      const noResultsHeight = noResults.getBoundingClientRect().height;
+      map.style.top =
+        `calc(var(--yxt-maps-mobile-results-header-height) + ${noResultsHeight}px)`;
+      const headerFooterHeight = 
```

Comment:
I think it's a bit confusing because I'd expect it to be an addition but because it's a string interpolation it has to be a subtraction. Was this moved out because the line was too long? I am of the camp that long strings are fine, what do you think?

Reply:
hmm, I can split the string up into multiple lines. I'm fine with long strings but not long lines

Reply:
maybe I would have a different opinion if my monitor was bigger